index.html

```diff
@@ -427,10 +427,59 @@
         <p>When subscribers receive a content distribution request with the<samp>X-Hub-Signature</samp> header specified, they SHOULD recompute the signature with the shared secret using the same method (provided in the <samp>X-Hub-Signature</samp> header) as the hub. If the signature does not match, subscribers MUST still return a 2xx success response to acknowledge receipt, but locally ignore the message as invalid. Using this technique along with HTTPS [[!RFC2818]] for subscription requests enables simple subscribers to receive authenticated notifications from hubs without the need for subscribers to run an HTTPS [[!RFC2818]] server.</p>
         <p>Please note however that this signature only ensures that the payload was not forged. Since the notification also includes headers, these should not be considered as safe by the subscriber, unless of course the subscriber uses HTTPS [[!RFC2818]] callbacks.</p>
       </section>
+    </section>
+  </section>
+
+  <section id="security-considerations">
+    <h2>Security Considerations</h2>
+    <p>Here is a summary of security considerations. It is important to note that WebSub is a server to server protocol which relies only on HTTP. It is strongly recommended to use HTTPS for all requests.</p>
+    <h3>Discovery</h3>
+    <p>There is no specific security consideration for discovery.</p>
+    <h3>Subscriptions</h3>
+    <p>First, subscribers SHOULD always favor the HTTPS url for hubs (even if the url is advertised as HTTP). Then the subscribers SHOULD use capability URLs for the callbacks, as well as make them available via HTTPS. Finally, subscribers SHOULD use a <samp>hub.secret</samp> when subscribing to allow signature of the content distribution. Hubs SHOULD enforce short lived <samp>hub.lease_seconds</samp>(10 days is a good default). When performing intent verification, the hub SHOULD use a random, single use <samp>hub.challenge</samp>.</p>
```

Comment:
Are you suggesting that there might be some out-of-band knowledge about a hub URL supporting https? If that's the case, then I think that should be made explicit. Otherwise it kind of sounds like you're suggesting a subscriber attempt an https connection even if the hub URL discovered is http.